msctl

```diff
@@ -391,12 +391,20 @@ getValue() {
   if [ -e "$1" ]; then
     # Find the key/value combo.
     KEY=$($PERL -ne '
-      $_ =~ s/[\x22\x27\r]//;
-      if ($_ =~ /^('$2')=.*$/i) { print lc $1; }
+      # Remove single and double quotes plus CR and LF.
+      $_ =~ s/[\x22\x27\r\n]//g;
+      # Remove comments.
+      $_ =~ s/\x23.*//;
```

Comment:
I wonder if this would be better as:
```
$_ =~ s/^\s*\x23.*//;
```

Let's say a someone has modified one of the URLs with a `#`, which isn't uncommon.  The code will currently assume everything after the `#` is a comment and will strip it.  If we require comments to be at the beginning of the line (with or without spaces first), we can get around this problem.  Shell scripts might allow a comment on the line, but I don't know if it is worth catching every possible configuration.